src/main/java/com/avispl/symphony/dal/communicator/epson/projector/EpsonProjectorCommunicator.java

```diff
@@ -0,0 +1,757 @@
+                    if(success){
+                        disconnect();
+                    }
+                    break;
+                case "A/V Mute":
+                    toggleSwitch("0".equals(value) ? MUTE_OFF : MUTE_ON, property, value);
+                    break;
+                case "Freeze":
+                    toggleSwitch("0".equals(value) ? FREEZE_OFF : FREEZE_ON, property, value);
+                    break;
+                case "Image settings#Brightness":
+                    setNumericControl(SET_BRIGHTNESS, property, (int) Float.parseFloat(value));
+                    break;
+                case "Image settings#Contrast":
+                    setNumericControl(SET_CONTRAST, property, (int) Float.parseFloat(value));
+                    break;
+                case "Image settings#Density":
+                    setNumericControl(SET_DENSITY, property, (int) Float.parseFloat(value));
+                    break;
+                case "Image settings#Tint":
+                    setNumericControl(SET_TINT, property, (int) Float.parseFloat(value));
+                    break;
+                case "Image settings#Sharp":
+                    setNumericControl(SET_SHARP, property, (int) Float.parseFloat(value));
+                    break;
+                case "RGB#Red":
+                    setNumericControl(SET_RED, property, (int) Float.parseFloat(value));
+                    break;
+                case "RGB#Green":
+                    setNumericControl(SET_GREEN, property, (int) Float.parseFloat(value));
+                    break;
+                case "RGB#Blue":
+                    setNumericControl(SET_BLUE, property, (int) Float.parseFloat(value));
+                    break;
+                case "Image settings#Color temperature":
+                    setNumericControl(SET_IMG_TEMPERATURE, property, (int) Float.parseFloat(value));
+                    break;
+                case "Image color mode":
+                    setImageColorMode(value);
+                    break;
+                default:
+                    logger.warn(String.format("Control operation %s is not supported.", value));
+                    break;
+            }
+        } finally {
+            controlOperationsLock.unlock();
+        }
+    }
+
+    @Override
+    public void controlProperties(List<ControllableProperty> list) throws Exception {
+        if (CollectionUtils.isEmpty(list)) {
+            throw new IllegalArgumentException("Controllable properties cannot be null or empty");
+        }
+
+        for(ControllableProperty controllableProperty: list){
+            controlProperty(controllableProperty);
+        }
+    }
+
+    @Override
+    public List<Statistics> getMultipleStatistics() throws Exception {
+        ExtendedStatistics statistics = new ExtendedStatistics();
+        if(isValidControlCoolDown() && localStatistics != null){

```

Comment:
Not thread safe - localStatistics is accessed without lock here while in all other places it is under lock.